diagnose_customer_currency.py

```
@@ -0,0 +1,143 @@
+                            print(f"      Price: {price.id}")
+                            print(f"      Amount: {price.unit_amount / 100:.2f} {price.currency.upper()}")
+
+                    # Check if it's EUR and active/trialing
+                    if sub.currency.lower() == 'eur' and sub.status in ['active', 'trialing', 'past_due']:
+                        print(f"\n      ⚠️  PROBLEM: Active EUR subscription!")
+                        print(f"      This is blocking USD subscriptions.")
+                        print(f"\n      To fix, cancel this subscription:")
+                        print(f"      >>> Cancel in Stripe Dashboard or run fix below")
+
+                        # Offer to cancel
+                        response = input(f"\n      Cancel this EUR subscription? (yes/no): ").strip().lower()
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)